frame/pablo/src/lib.rs

```diff
@@ -591,6 +600,18 @@ pub mod pallet {
 				);
 				if result.is_ok() {
 					weight += 1;
+					if let Some(updated_twap) = TWAPState::<T>::get(pool_id) {
+						if let Ok(currency_pair) = Self::currency_pair(pool_id) {
+							Self::deposit_event(Event::<T>::TWAPUpdated {
+								pool_id,
+								timestamp: updated_twap.timestamp,
+								twaps: BTreeMap::from([
+									(currency_pair.base, updated_twap.base_twap),
```

Comment:
just to double check, that is possible that base_twap current == previous base_twap ?

Reply:
so question is if there was no trades, then should we see events - i think not.